### PR TITLE
restructure

### PR DESCRIPTION
**Pull request by kejuntong**
## Description
- restructure the ui folders and files

image

- group common components together and reuse

image

- after the change, need to adjust config to run the app:

image

**No functional change.** 

## Type of change
- [ ] Chore (copy changes, version bump etc...)
- [ ] Bug fix (fixes an issue)
- [x] Refactor (modifies existing functionality)
- [ ] New feature (adds functionality)

### CODE REVIEW

File: lib/ui/component/alert_options.dart [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                  style: Theme.of(context).textTheme.bodyLarge),
+              Switch(
+                value: widget.options.isRepeat,
+                onChanged: (enabled) {
+                  widget.optionsOnChange(widget.options.setIsRepeat(enabled));
+                },
+              ),
+            ],
+          ),
+          Offstage(
```
Comment (kejuntong) [inline lib/ui/component/alert_options.dart:38]: could probably wrap all offstages together

**Review by WenjiuWang — APPROVED** [review]
tested locally. lgtm

**Review by q58wu — APPROVED** [review]
LGTM!

**Comment by q58wu** [comment]
thanks for addressing the folder structure!

**Pull request merged**